src/components/VaultCreationDialog.tsx

```diff
@@ -0,0 +1,90 @@
+import { Dialog, Transition } from '@headlessui/react';
+import { useSetAtom, useAtomValue } from 'jotai';
+import { Fragment } from 'react';
+import { walletUiHrefAtom } from 'store/app';
+import { ViewMode, viewModeAtom } from 'store/vaults';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+};
+
+const VaultCreationDialog = ({ isOpen, onClose }: Props) => {
+  const walletUrl = useAtomValue(walletUiHrefAtom);
+  const setViewMode = useSetAtom(viewModeAtom);
+
+  const goToWallet = () => {
+    window.open(walletUrl, 'wallet');
```

Comment:
consider using the third param to specify ideal window dimensions

Reply:
I think it's preferable to just let it open in a new tab if possible.